### DISCUSSION TITLE
whats the difference between installing git via laragon quick install vs installing it standalone

### DISCUSSION THREAD

**Discussion by mbenic** [leokhoa/laragon#921] [General]
hi 
im wondering what is the difference between installing git , node.js , vs code by using the quick install option in laragon 
as opposed to downloading all these programs from the internet and installing them separately to laragon
can someone please explain ?
thanks

**Comment by husnilkhatimi** [comment]
By installing using Laragon setup, it can becomes portable and isolated environment because all the tools will be installed in Laragon own bin folder.

For me, I prefer to install all those tools separately using their own latest installer. So easier to update each one and can use their command system-wide.

**Comment by mbenic** [comment]
if you go to Tools->Path-> and 'Add Laragon to Path'  doesnt this enable you to use all those applications system-wide?
